1.6.0 (#25)
MPI-EVA-Archaeogenetics/Autorun_eager: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [1.6.0] - XX/XX/XXXX

### `Added`

- `conf/Autorun.config`: Use hard links when publishing results, instead of copying files.
- `scripts/create_poseidon_release.sh`: New script to create large releases of the entire TF processed data in Poseidon format.

> - The following scripts now use pyPandoraHelper to infer Site_ID from Ind_ID:

Author: ```suggestion
- The following scripts can now infer Site_ID of varied lengths from the Ind_ID (pyPandoraHelper):
```

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [1.6.0] - XX/XX/XXXX

### `Added`

- `conf/Autorun.config`: Use hard links when publishing results, instead of copying files.
- `scripts/create_poseidon_release.sh`: New script to create large releases of the entire TF processed data in Poseidon format.
- The following scripts now use pyPandoraHelper to infer Site_ID from Ind_ID:
  - `scripts/clear_results.sh`
  - `scripts/clear_work_dirs.sh`
  - `scripts/ethical_sample_scrub.sh`
  - `scripts/run_Eager.sh`
  - `scripts/update_poseidon_packages.sh`

> - The following scripts now use rPandoraHelper to infer Site_ID from Ind_ID:
>   - `scripts/prepare_eager_tsv.R`
>   - `scripts/fill_in_janno.R`
> - Now compatible with Pandora Site IDs longer than 3 letters.

Author: ```suggestion
- The following scripts can now infer Site_ID of varied lengths from the Ind_ID (rPandoraHelper):
  - `scripts/prepare_eager_tsv.R`
  - `scripts/fill_in_janno.R`
```
